### ISSUE TITLE
Updating TCK instructions

### ISSUE THREAD

**Issue by arthurdm** [eclipse/microprofile-open-api#398]
We have [this page](https://github.com/eclipse/microprofile-open-api/blob/master/tck/running_the_tck.asciidoc) outlining how to run the TCK, but that's quite old now.  I am wondering if anyone is using different instructions, etc.    If so, we should update that page.

Also - if there's anything specific to running it against SmallRye (CC @EricWittmann @MikeEdgar), we could have a new section on that page for it - which could serve as a sample for other implementations that want to add specific instructions there too.

**Comment by EricWittmann** [comment]
The SmallRye OpenAPI library isn't a full MP implementation (obviously) so we don't technically run against the TCK in an "official" way.  Instead, I have created a custom junit layer that executes all of the TCK tests without spinning up a real environment.  It's very useful but does not constitute a real execution of the TCK.  That happens in places like Quarkus and Thorntail(rip).

I'm not against documenting what's in SmallRye OpenAPI in some way, but am not convinced it would be appropriate to do that in MP.  Also, the documentation would be:  `mvn test` :)

**Comment by arthurdm** [comment]
Fair point. =)       Are the instructions still applicable for something like Quarkus?

**Comment by EricWittmann** [comment]
Honestly I'm not sure - @kenfinnigan is there someone on the Quarkus side who can easily answer this?  :)
